include/sys/bus.h

```diff
@@ -210,6 +210,9 @@ static inline void bus_release_resource(device_t *dev, res_type_t type,
   BUS_METHODS(idev->parent).release_resource(idev, type, r);
 }
 
+/* System-wide current pass number. */
+extern pass_num_t current_pass;
```

Comment:
I think in current use scenario you can get rid of it.

Reply:
Done.